ivadomed/utils.py

```diff
@@ -562,6 +586,40 @@ def structureWise_uncertainty(fname_lst, fname_hard, fname_unc_vox, fname_out):
     nib.save(nib_avgUnc, fname_avgUnc)
 
 
+def multiclass_dice_score(im1, im2):
+    im1 = np.asarray(im1[0, ]).astype(np.bool)
+    im2 = np.asarray(im2[0, ]).astype(np.bool)
+
+    dice_per_class = 0
+    n_classes = im1.shape[0]
```

Comment:
I thought the "class dimension" was the last dimension --> are you doing some reshaping at some point?

Reply:
The shape format is batch_size, n_classes, height, depth, width. There used to be a squeeze before metrics which made the class number be in first place if the batch_size was 1, but I removed it. n_classes should be in im1.shape[1]! Thanks for that!